### ISSUE TITLE
Can't scroll past a horizontal scrolling area on touch devices

### ISSUE THREAD

**Issue by nickyblissAviva** [malihu/malihu-custom-scrollbar-plugin#529]
We have a number of areas on our sites eg. tables of data that need to be horizontally scrollable. The problem we are seeing is that when using the plugin it stops the user being able to scroll down the page unless they touch outside of the horizontally scrolling area.

I have been able to fix this for some touch devices by using the following css
`.mCustomScrollBox.mCSB_horizontal {
	touch-action: pan-y pinch-zoom;
}`

Unfortunately this does not work for iOS Safari.

**Comment by andi-b** [comment]
Yeah I'm having this issue too. By the looks of it, when swiping to scroll down the page, the horizontal scroll is slightly moving. Could almost do a sensitivity option

**Comment by PaperCoder** [comment]
Have a look at #536

**Comment by nickyblissAviva** [comment]
We managed to fix Chrome with some css changes but iOS still fails as it does not support css touch-action

The following changes were required for the css to fix Chrome
```
.mCustomScrollbar,
.mCustomScrollbar.mCS_touch_action,
.mCustomScrollbar.mCS_touch_action .mCS-aviva.mCSB_scrollTools {
	-ms-touch-action: auto;
	touch-action: auto;
}
.mCustomScrollBox {
	-ms-touch-action: pan-x pinch-zoom;
	touch-action: pan-x pinch-zoom;
}
.mCustomScrollBox.mCSB_vertical_horizontal {
	-ms-touch-action: pinch-zoom;
	touch-action: pinch-zoom;
}
.mCustomScrollBox.mCSB_horizontal {
	-ms-touch-action: pan-y pinch-zoom;
	touch-action: pan-y pinch-zoom;
}
```

**Comment by PaperCoder** [comment]
I've testet it with iOS 10.3.2 and everything works fine. Which version do you use?

**Comment by nickyblissAviva** [comment]
Various versions of iOS and iOS devices including 10.2 (I can't get hold of the 10.3.2 testing device today)

The top table on this test page has the issue. Unless you touch the edge of the page outside of the horizontally scrolling area you can't scroll past the first table on the page. We had the same behavior in Chrome until we adjusted the touch-action settings in the css.
https://standards.aviva.com/framework/test/tables.html

**Comment by PaperCoder** [comment]
I can't reproduce this problem. Just tested your url with Safari on an IPhone 6 and iOS version 9 on BrowserStack and everything is working fine.

**Comment by nickyblissAviva** [comment]
There is definitely an issue as we have got multiple physical devices from different teams around the world showing the same issue. That they can't scroll past the table.
I would never 100% trust BrowserStack.

**Comment by PaperCoder** [comment]
I tested it now with a real iPhone 10.2.1 and you're right. Your example is not working. But I'll tested afterwards my version where I used customScrollbar 3.1.5 and the deleted line in CSS and everything works fine.

**Comment by nickyblissAviva** [comment]
Thats very odd. Will have to try that Monday when I have access to the devices again.

**Comment by nickyblissAviva** [comment]
@PaperCoder 

Have managed to procure an iPhone in the office and tested on our test environment and changing the CSS to 
```
.mCustomScrollbar,
.mCustomScrollbar.mCS_touch_action,
.mCustomScrollbar.mCS_touch_action .mCS-aviva.mCSB_scrollTools {
	-ms-touch-action: auto;
	    touch-action: auto;
}

.mCustomScrollBox {
	-ms-touch-action: pan-x;
	    touch-action: pan-x;
}

.mCustomScrollBox.mCSB_vertical_horizontal {
	-ms-touch-action: auto;
	    touch-action: auto;
}

.mCustomScrollBox.mCSB_horizontal {
	-ms-touch-action: pan-y;
	    touch-action: pan-y;
}
```

Seems to ensure that the iPhone works, Windows Phone is also happier when touch-action is set too as without it at all, it got a bit sluggish when scrolling horizontally.
We'll need to perform our full browser testing now but its looking hopeful that we have got a solution.
